### PR TITLE
Transform all java classes to kotlin

### PR DESCRIPTION
**Pull request by sergiandreplace**
Transformed all classes in the project from Java to Kotlin.

I run into several issues with nullable values and callbacks, so I had to do quite a lot of manual fixing.

For you convenience, the first commit contains only the automatic changes, the second one all the fixes, and the third one just some autoformatting.

There is a lot of room for improvement in a Kotlin way, but I prefer to left this for future PR and keep this one as simple as possible.

tenor-73625478

### CODE REVIEW

File: library/src/main/java/com/sloydev/preferator/Preferator.kt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package com.sloydev.preferator
+
+import android.content.Context
+import android.content.Intent
+
+object Preferator {
+  fun launch(context: Context) {
```
Comment (Sloy) [inline library/src/main/java/com/sloydev/preferator/Preferator.kt:8]: This function (and the no-op version) should be annotated with `@JvmStatic` to keep Java compatibility.

**Comment by Sloy** [comment]
Thanks a lot! The commit separation really helped. Let my try it out locally with a bigger project to make sure everything still works

File: library/src/main/java/com/sloydev/preferator/Preferator.kt (hunk 1 above)
Comment (sergiandreplace) [reply library/src/main/java/com/sloydev/preferator/Preferator.kt:8]: added!

**Review by Sloy — APPROVED** [review]
Great! Now it works! :)

**Pull request merged**